distutils/tests/test_check.py

```diff
@@ -180,14 +174,7 @@ def foo():
             cmd = check(dist)
             cmd.check_restructuredtext()
             msgs = cmd._check_rst_data(rest_with_code)
-            if pygments is not None:
-                assert len(msgs) == 0
-            else:
-                assert len(msgs) == 1
-                assert (
-                    str(msgs[0][1])
-                    == 'Cannot analyze code. Pygments package not found.'
-                )
```

Comment:
I don't see pygments being use anywhere or even installed as a transitive dependency. Is that just old code ?